src/js/utils/process-json.js

```diff
@@ -61,6 +61,32 @@ function convertDate( date ) {
 }
 
 /**
+ * Prepares mortgage delinquency data for Highcharts.
+ *
+ * @param {Number} data - Raw JSON from mortgage-performance API
```

Comment:
linting: the reported parameters are off. "Expected JSDoc for 'datasets' but found 'data'" and "Missing JSDoc for parameter 'timeSpan'"